index.js

```diff
@@ -48,7 +48,6 @@ function createMainWindow() {
 		backgroundColor: storage.get('darkMode') ? '#192633 ' : '#fff',
 		webPreferences: {
 			preload: path.join(__dirname, 'browser.js'),
-			nodeIntegration: false,
```

Comment:
Comment it out instead with a comment about enabling it again when the Electron preload ticket is fixed.